mapeditor/windownewmap.h

```diff
@@ -73,7 +74,7 @@ class WindowNewMap : public QDialog
 		{3, {CMapHeader::MAP_SIZE_XLARGE,	CMapHeader::MAP_SIZE_XLARGE}},
 		{4, {CMapHeader::MAP_SIZE_HUGE,		CMapHeader::MAP_SIZE_HUGE}},
 		{5, {CMapHeader::MAP_SIZE_XHUGE,	CMapHeader::MAP_SIZE_XHUGE}},
-		{6, {CMapHeader::MAP_SIZE_GIANT,	CMapHeader::MAP_SIZE_GIANT}},
+		{6, {CMapHeader::MAP_SIZE_GIANT,	CMapHeader::MAP_SIZE_GIANT}}
```

Comment:
why is this change needed?

Reply:
Probably not needed, I have tried to add new map size "I (999x999)" - MAP_SIZE_INSANE accross game and in CRmgTemplate.cpp is not "," at the end so I did it same after reverting my changes.
https://github.com/vcmi/vcmi/blob/543e241769fa8986080031d59da2061f1788dfd9/lib/rmg/CRmgTemplate.cpp#L903

Reply:
And here I think is valid approach without traling comma at the end.

Reply:
anything that accepts initializer list allows trailing comma since C++11 (enum cases as well btw). It's better to add it because when you add new entries in the future, you won't have to touch the last existing line.

Reply:
Well, it was only place where it was. All three sections in that file have now same standard. And I don't think there is currently any plan in adding new map sizes / more players.

![image](https://github.com/user-attachments/assets/6a356b6f-a9f8-4021-9a0e-fc86de6c7824)

Reply:
I suggest to revert this comma change. And you could also remove extra tab in MAP_SIZE_HUGE.

Reply:
This is how it looks after extra tab removal.
![image](https://github.com/user-attachments/assets/53d5dea9-4e03-4c0c-9854-02cfdee8116d)

Reply:
different tab width settings :) then it should be separated with spaces to avoid misalignment

Reply:
OK. 
![image](https://github.com/user-attachments/assets/73e32641-db8f-4a0f-b903-09e5462dd5dd)